data-prepper-plugins/otel-trace-source/src/test/java/org/opensearch/dataprepper/plugins/source/oteltrace/OTelTraceSourceTest.java

```diff
@@ -852,6 +877,61 @@ void gRPC_request_writes_to_buffer_with_successful_response() throws Exception {
         assertThat(actualBufferWrites, hasSize(1));
     }
 
+    @Test
+    void gRPC_with_auth_request_writes_to_buffer_with_successful_response() throws Exception {
```

Comment:
We should have a test like this which includes the custom path. It should verify that the path is secured.